docs/dev-notes.md

```diff
@@ -0,0 +1,83 @@
+Make
+####
+
+We're currently using GNU Make as the primary tool for builds. We're
+using it as a fairly unsophisticated driver for directory specific
+scripts.
+
+If you run just plain
+
+    make
+
+you should see a list of goals that you can run, for example
+
+    make build-spamgourmet-clone
+
+to build the docker container or
+
+    make all
+
+to run the standard set of goals expected during development. If you
+have a container built then
+
+    make shell
+
+should get you a shell inside a docker container with all of
+spamgourmet set up and with all of the spamgourmet code visible to
+allow you to test code you are working on.
+
+GitHub setup
+############
```

Comment:
Viewing [this file](https://github.com/spamgourmet/spamgourmet/blob/4990e027526f7651e5e17ffabe8f80a2bc0702b3/docs/dev-notes.md) in GitHub does not render too well. I like to use the `markdownlint` extension in VS Code to format the files.


```suggestion
## GitHub setup
```